CHANGELOG.md

```diff
@@ -1,216 +1,67 @@
-| <!--PWA-922-->Updated the Venia splash image shown when JavaScript is disabled in the browser.                                                         | [3355][]         |
-| <!--PWA-983-->Moved all project `CartPage/.../*.gql.js` files from `venia-ui` to the `peregrine/talons` directory.                                     | [3457][]         |
-| <!--PWA-984-->Moved all project `CheckoutPage/.../*.gql.js` files from `venia-ui` to the `peregrine/talons` directory.                                 | [3441][]         |
-| <!--PWA-1244-->Refactor all queries that use category or product `url_suffix` to use the `storeConfig` suffix.                                         | [3393][]         |
-| <!--PWA-1278-->Added keyboard accessiblity to the `MegaMenu` component. Users can now navigate through all the MegaMenu links with their keyboards.    | [3319][]         |
-| <!--PWA-1702-->Enabled trusted extension vendors to change code outside their namespace, namely within `@magento/[packages]`.                          | [3266][]         |
-| <!--PWA-1790-->Enabled Page Builder `Row` appearances (Full Width, Full Bleed) to work as expected on the Venia storefront.                            | [3221][]         |
-| <!--PWA-1868-->Implemented new GraphQL caching header that Venia and other storefronts must send to GraphQL.                                           | [3278][]         |
-| <!--PWA-1909-->Updated CSS Module source maps to make CSS `className` easy to find and change during development.                                      | [3407][]         |
-| <!--PWA-1928-->Added extensibility point (new target) for new Page Builder content types: `setContentTypeConfig`                                       | [3307][]         |
-| <!--PWA-1933-->Added two new status to Order History page: `received` (order submitted but not processed) and `rejected` (failure when placing order). | [3431][]         |
-| <!--PWA-1964-->Updated Workbox packages to version `6.2.4` to fix the backwards compatibility issue we had with our previous version `6.0.2`.          | [3378][]         |
+-  **Added new PWA Tailwind theming to Venia Header**: Refactored the Venia site `Header` component to use our new [Tailwind](https://tailwindcss.com/) theming framework. The `Header` component is the first component to use our theming framework. Other components will follow in the coming releases. GitHub PR: [3472][].
+-  **Added PWA Studio metapackages**: These metapackages establish a new way to add additional features to support your PWA modules. Future PWA Studio features will depend on one of these metapackages being present in your app. For more details, see the [PWA Magento Open Source metapackage](https://github.com/magento-commerce/magento2-pwa) and [PWA Adobe Commerce metapackage](https://github.com/magento-commerce/magento2-pwa-commerce) GitHub PRs: [4][], [1][].
```

Comment:
>Future PWA Studio features will depend on one of these metapackages being present in your app.

Needs to be updated (and perhaps strongly advertised) that this release (12.1) requires this metapackage be installed now.

Reply:
Got it. Just pushed updates.